code/email/CaptureMailer.php

```diff
@@ -13,14 +13,9 @@ class CaptureMailer extends Mailer {
 	 *
 	 * @var type
 	 */
-	public static $capture_emails = true;
+	private static $capture_emails = true;
```

Comment:
With changing this to a private static, you need to change all references to self::$capture_emails to be a Config::get() reference, otherwise it can never be changed from the outside. 

Alternatively, change this to `public $captureEmails = true;` so it can be set via the injector, and change self::$capture_emails to $this->captureEmails